docs/api/tools/block-explorer/block-view.md

```diff
@@ -0,0 +1,56 @@
+# View Blocks
+
+![Block view!](../../../assets/images/block-tx.png "View blocks on zkSync")
+
+The block explorer home screen, by default, displays the 5 most recent blocks in the chain. 
+View all blocks for a more complete overview of recent blocks, or click on a specific block number for details on that block.
+The blocks page is also accessible via the top menu.
+
+![Access the block page](../../../assets/images/block-menu.png "Blocks menu")
+
+## View all blocks
+
+The view all blocks page shows updated details from the most recent blocks. Use the pagination to navigate to the next page.
+
+However, if you have already left the homepage, you can always search for any Ethereum, DAI, LINK, wBTC, and USDC block by typing in the block number.
```

Comment:
I don't think we need to mention tokens here. You can enter a number and you'll search for that specific block regardless of the tokens